device/src/lib.rs

```diff
@@ -5,11 +5,11 @@ extern crate alloc;
 
 pub mod device_config;
 pub mod esp32_run;
+#[cfg(feature = "v2")]
+pub mod graphics;
```

Comment:
:pick: I actually liked it being `st7789` since the graphics is tightly bound to the screen dimensions. If we were to make a second edition with a different screen size etc where would we put it?

Reply:
We could pass the mipidsi Model and implement Graphics for a different display model